docs/UserGuide.adoc

```diff
@@ -43,9 +44,12 @@ image::Ui.png[width="790"]
 e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 .  Some example commands you can try:
 
-* *`(ab)list`* : lists all contacts
-* **`(ab)add`**` n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : adds a contact named `John Doe` to the Address Book.
```

Comment:
You might have left out the (ab)list command in the UG.

Reply:
So (ab)list just lists all contacts then (ab)find allows users to list just specific contacts? Will it be considered overlapping commands though? O:

Reply:
Okay I added it back in already!